application/src/main/java/org/opentripplanner/transit/api/request/RegularStopRequestBuilder.java

```diff
@@ -0,0 +1,28 @@
+package org.opentripplanner.transit.api.request;
+
+import org.locationtech.jts.geom.Envelope;
+
+public class RegularStopRequestBuilder {
+
+  private Envelope envelope;
+  private String agency;
+  private boolean filterByInUse = false;
+
+  protected RegularStopRequestBuilder(Envelope envelope) {
+    this.envelope = envelope;
+  }
+
+  public RegularStopRequestBuilder withFeed(String agency) {
```

Comment:
Here we still have `withFeed`.

Reply:
Discussed offline. Feed is actually the right thing.